src/corehost/cli/deps_resolver.cpp

```diff
@@ -597,7 +598,7 @@ void deps_resolver_t::resolve_additional_deps(const hostpolicy_init_t& init)
         return;
     }
 
-    pal::string_t additional_deps_serialized = init.additional_deps_serialized;
+    pal::string_t additional_deps_serialized = args.additional_deps_serialized;
```

Comment:
Should additional-deps affect just the main app, or all plugins?

Reply:
For component dependency resolution the `args.additional_deps_serialized` will always be an empty string. Look at hostpolicy.cpp line 508 where the args are initialized.

Reply:
I added a test case for this to make sure.